packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFNavigationDelegateHostApi.m

```diff
@@ -26,15 +27,86 @@ - (long)identifierForDelegate:(FWFNavigationDelegate *)instance {
 
 - (void)didFinishNavigationForDelegate:(FWFNavigationDelegate *)instance
                                webView:(WKWebView *)webView
-                                   URL:(NSString *)URL {
+                                   URL:(NSString *)URL
+                            completion:(void (^)(NSError *_Nullable))completion {
   [self didFinishNavigationForDelegateWithIdentifier:@([self identifierForDelegate:instance])
                                    webViewIdentifier:
                                        @([self.instanceManager
                                            identifierWithStrongReferenceForInstance:webView])
                                                  URL:URL
-                                          completion:^(NSError *error) {
-                                            NSAssert(!error, @"%@", error);
-                                          }];
+                                          completion:completion];
+}
+
+- (void)didStartProvisionalNavigationForDelegate:(FWFNavigationDelegate *)instance
+                                         webView:(WKWebView *)webView
+                                             URL:(NSString *)URL
+                                      completion:(void (^)(NSError *_Nullable))completion {
+  [self didStartProvisionalNavigationForDelegateWithIdentifier:@([self
+                                                                   identifierForDelegate:instance])
+                                             webViewIdentifier:
+                                                 @([self.instanceManager
+                                                     identifierWithStrongReferenceForInstance:
```

Comment:
It would help the readability of all of these calls if you pulled this identifier lookup out into a local variable.

Reply:
Ah, good suggestion. I sometimes forget that the auto formatter can make your code look worse sometimes. done